azure-iot-device/azure/iot/device/provisioning/pipeline/config.py

```diff
@@ -0,0 +1,17 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import logging
+from azure.iot.device.common.pipeline.config import BasePipelineConfig
+
+logger = logging.getLogger(__name__)
+
+
+class ProvisioningPipelineConfig(BasePipelineConfig):
+    """A class for storing all configurations/options for IoTHub clients in the Azure IoT Python Device Client Library.
```

Comment:
nit: shouldn't this comment specify this is the COnfig class specifically for Provisioning?